docs/bn/week02/02-2.md

```diff
@@ -0,0 +1,258 @@
+---
+lang-ref: ch.02-2
+lecturer: Yann LeCun
+title: Computing gradients for NN modules and Practical tricks for Back Propagation
```

Comment:
The title needs translating.

Reply:
Thanks, @Atcold. I've translated the title now.